package.json

```diff
@@ -1,19 +1,18 @@
 {
-  "author": "Shinichi Tomita <shinichi.tomita@gmail.com>",
```

Comment:
why remove the author and change the description?
I think the main author is still the same and the package still does the same thing